### ISSUE TITLE
About the better recall 

### ISSUE THREAD

**Issue by wmn931201** [sergiomsilva/alpr-unconstrained#65]
Hi,@sergiomsilva, I have learned your updated code, I find that there is no change in the framework of wpod-net  and data augmentation. the main change is decoupling model creation from training script and using threads for data generation. also, batch_size, learning_rate and some small tricks have changed. Do these changes improve the recall?  Thank you very much!

**Comment by sergiomsilva** [comment]
No, it was just a fine-tuning that I made myself in the original model. Regards.

**Comment by wmn931201** [comment]
Thank you ! @sergiomsilva . I fine-tuned my own model  based on the original model you published  in September last year. However, There are many false detections in the test phase. There are no such false detections on the original model. Take the liberty to ask, what do you think will be caused?

**Comment by sergiomsilva** [comment]
This looks like overfitting. Did you fine-tune using only data from your scenario? I recommend you use all the data that we used for training plus your data. You should also check if the annotations you made are correct. Regards

**Comment by wmn931201** [comment]
The data I used for training consist of AOLP, Cars and data from my scenario. SSIG is difficult to get. and there should be no errors in my annotations. Thus, I think the false detection is  very confusing. I take the liberty of asking if you can share SSIG dataset to me. Thanks you very much!

**Comment by sergiomsilva** [comment]
@wmn931201 I cant, all datasets used need legal agreements.

**Comment by wmn931201** [comment]
OK，I know. @sergiomsilva, Thank you very much!
